packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -7,7 +7,7 @@
 
 import throat from 'throat';
 import type {JestEnvironment} from '@jest/environment';
-import {type JestExpect, jestExpect} from '@jest/expect';
```

Comment:
this was probably fine btw as it's not exposed after bundling type definitions (it's internal)

Reply:
It's fine to do the build using a higher version of ts, as long as the output works older versions 😀